mainnet-contracts/src/PufferVaultV5.sol

```diff
@@ -288,12 +290,23 @@ contract PufferVaultV5 is
         if (assets > maxAssets) {
             revert ERC4626ExceededMaxWithdraw(owner, assets, maxAssets);
         }
+        VaultStorage storage $ = _getPufferVaultStorage();
+
+        uint256 assetsWithFeeIncluded = _assetsWithFee(assets, $.exitFeeBasisPoints + $.treasuryExitFeeBasisPoints);
+
+        uint256 treasuryFee = _feeOnRaw(assetsWithFeeIncluded, $.treasuryExitFeeBasisPoints);
 
-        _wrapETH(assets);
+        uint256 shares = super.previewWithdraw(assetsWithFeeIncluded);
```

Comment:
previewWithdraw is doing `_assetsWithFee(amount, totalFees)` inside of it. 

and the amount is already `assetsWithFeeIncluded`. It looks like we would burn more pufETH from the user than we are supposed to?